src/main/java/org/mule/extension/api/runtime/ContentMetadata.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.runtime;
+
+/**
+ * Provides metadata regarding the content of a {@code MuleMessage} in the
+ * context of an operation execution. As such, it distinguishes between the
+ * metadata of the content that enters the operation and the one that goes
+ * out of it. Those metadata values are expressed as instances of
+ * {@link ContentType}.
+ * <p>
+ * It also accounts for the fact that in certain cases, it's the operation
+ * or even the runtime the ones which calculate the output {@link ContentType}.
```

Comment:
it's the operation or even the runtime the ones which calculate the output > is the operation or event the runtime the one which calculates the output?